test/drop_test/action.py

```diff
@@ -135,23 +131,18 @@ def __init__(self, peers: typing.List[str], path: str):
         self._fd: typing.Any = None
 
     async def run(self, drop: ffi.Drop):
-        # save to object in order to increase the lifetime of the file until GC collects it and closes the file
-        self._fd = open(self._path, "r")
-
-        fo = self._fd.fileno()
-
         for peer in self._peers:
             UUIDS_LOCK.acquire()
 
-            xfid = drop.new_transfer_with_fd(peer, self._path, fo)
+            xfid = drop.new_transfer_with_fd(
+                peer, self._path, f"content://cached{self._path}"
+            )
             UUIDS.append(xfid)
 
             UUIDS_LOCK.release()
 
     def __str__(self):
-        return (
-            f"MultipleNewTransfersWithSameFD({self._peers}, {self._path}, {self._fd})"
-        )
+        return f"MultipleNewTransfersWithSameFD({self._peers}, {self._path})"
```

Comment:
Isn't it worth just using `NewTransferWithFD` ? This function does nothing wit the `fd`s anymore

Reply:
Let me see. It might be a leftover

Reply:
Oh, I see now. So this reuses the FD multiple times and the `NewTransferWithFD` creates a new one each time

Reply:
A bool flag indicating to cache the fd and reuse it would be nice improvement imho

Reply:
Done